<ui version="4.0" >
 <class>PreferencePageSession</class>
 <widget class="QWidget" name="PreferencePageSession" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>189</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>PreferencePageSession</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="0" column="0" >
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <layout class="QGridLayout" name="gridLayout" >
       <item row="1" column="0" >
        <widget class="QLineEdit" name="_directory" >
         <property name="minimumSize" >
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="readOnly" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QToolButton" name="_directoryBrowse" >
         <property name="text" >
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Directory with files</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <item>
        <widget class="QCheckBox" name="_readFrom" >
         <property name="text" >
          <string>Read from:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>37</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="_readFromFile" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editable" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" >
       <item>
        <widget class="QCheckBox" name="_writeTo" >
         <property name="text" >
          <string>Write to:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>50</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="_writeToFile" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editable" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1" >
    <spacer name="horizontalSpacer_3" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>7</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" name="horizontalLayout_3" >
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Request Time Out:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="_requestTimeout" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="wrapping" >
        <bool>false</bool>
       </property>
       <property name="frame" >
        <bool>true</bool>
       </property>
       <property name="accelerated" >
        <bool>false</bool>
       </property>
       <property name="minimum" >
        <number>100</number>
       </property>
       <property name="maximum" >
        <number>2000</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>28</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="_auto" >
       <property name="text" >
        <string>Auto</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" >
    <layout class="QHBoxLayout" name="horizontalLayout_4" >
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Request Queue Size:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="_queueSize" >
       <property name="minimumSize" >
        <size>
         <width>51</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>51</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="minimum" >
        <number>1</number>
       </property>
       <property name="maximum" >
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>second</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="2" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
